app/src/components/grid/gridContextMenu.tsx

```diff
@@ -1,58 +1,89 @@
-import { GridApi } from 'ag-grid-community';
-import { ContextMenuItem } from './gridTypes';
+import { GridApi } from "ag-grid-community";
+import { ContextMenuItem } from "./gridTypes";
 
 // region: Filters
-export const onFilterEqual = (gridApi: GridApi, params: any): ContextMenuItem => {
+export const onFilterEqual = (
+  gridApi: GridApi,
+  params: any,
+): ContextMenuItem => {
   const menuItem: ContextMenuItem = {
-    "name": "Filter equal",
+    name: "Filter equal",
     action: () => {
-
       let filterModel = gridApi.getFilterModel();
-      filterModel = filterModel === undefined ? {} : filterModel
+      filterModel = filterModel === undefined ? {} : filterModel;
 
       const selectedValue = params.value;
       let filterColumn = params.column.getColId();
 
       // Additional logic so that use can filter on the group columns.
       if (filterColumn === "ag-Grid-AutoColumn") {
-        filterColumn = gridApi.getRowGroupColumns()[params.node.level]
-          .getColDef()
-          .field
+        filterColumn = gridApi
+          .getRowGroupColumns()
+          [params.node.level].getColDef().field;
       }
-      filterModel[filterColumn] = {
-        type: "equals",
-        filter: selectedValue
+      if (typeof selectedValue === "number") {
+        const lowerBound = Math.round(selectedValue);
+        const upperBound = lowerBound + 1;
```

Comment:
In the future probably need to dynamically switch to +0.1 or + 0.01 based on the rounding.